tests/data/samplesheets/samplesheet_test1.csv

```diff
@@ -0,0 +1,5 @@
+sample,mlst_alleles,address
+sampleQ,https://raw.githubusercontent.com/phac-nml/gasnomenclature/add_tests/tests/data/reports/sampleF.mlst.json,
+sample1,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sample1.mlst.json,1.1.1
+sample2,https://raw.githubusercontent.com/phac-nml/gasnomenclature/add_tests/tests/data/reports/sample7.mlst.json,1.1.1
```

Comment:
As a note, once this PR is merged you'll have to update the URL here to point to `dev`. You can just directly commit that to the `dev` branch.